content/docs/sidebar.json

```diff
@@ -119,6 +119,24 @@
           }
         ]
       },
+      {
+        "slug": "data-management",
+        "source": false,
+        "children": [
+          "large-dataset-optimization",
+          "importing-external-data",
+          "managing-external-data"
```

Comment:
An alternative to consider (from https://github.com/iterative/dvc.org/issues/144#issuecomment-1206648437) which may be better since we don't love the product feature as-is now (I think):

> External Deps/Managing External Data: ... or make them only reachable via links.

Reply:
p.s. update the H1 in `importing-external-data.md`

Reply:
> or make them only reachable via links.

I like it better than having no solution for using data directly from the cloud, so I'm going to keep it in.